sdk/include/reactos/drivers/afd/shared.h

```diff
@@ -110,6 +110,8 @@ typedef struct _AFD_SEND_INFO_UDP {
     ULONG				AfdFlags;
     TDI_REQUEST_SEND_DATAGRAM		TdiRequest;
     TDI_CONNECTION_INFORMATION		TdiConnection;
+    PVOID				Address; // for CORE-17521
```

Comment:
Not sure where in this PR this variable is used.

Reply:
You are correct, it was added for an earlier version of the code which stored the address separately rather than using RemoteAddress. I don't have a build environment to test this on at the moment, but since they are not referenced at all by the code please feel free to remove them.

Reply:
removed using web editor